it/google-cloud-platform/build.gradle

```diff
@@ -0,0 +1,74 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins { id 'org.apache.beam.module' }
+applyJavaNature(
+        automaticModuleName: 'org.apache.beam.it.gcp',
+)
+
+description = "Apache Beam :: IT :: Google Cloud Platform"
+ext.summary = "Integration test utilities for Google Cloud Platform."
+
+dependencies {
+    implementation enforcedPlatform(library.java.google_cloud_platform_libraries_bom)
+    implementation project(path: ":sdks:java:core", configuration: "shadow")
+    implementation project(path: ":runners:google-cloud-dataflow-java")
+    implementation project(path: ":it:conditions", configuration: "shadow")
+    implementation project(path: ":it:truthmatchers", configuration: "shadow")
+    implementation library.java.slf4j_api
+    implementation library.java.vendored_guava_26_0_jre
+    implementation library.java.jackson_core
+    implementation library.java.jackson_databind
+    implementation 'org.apache.hadoop:hadoop-common:3.3.5'
+    implementation 'org.apache.avro:avro:1.11.1'
+    implementation 'org.apache.parquet:parquet-avro:1.12.0'
+    implementation 'org.apache.parquet:parquet-common:1.12.0'
+    implementation 'org.apache.parquet:parquet-hadoop:1.12.0'
+    implementation library.java.gax
+    implementation library.java.google_api_common
+    implementation library.java.protobuf_java_util
+    implementation library.java.protobuf_java
+    implementation 'org.threeten:threetenbp:1.6.8'
```

Comment:
Even though joda time is deprecated, it is what Beam uses. We should use it here too, and change as part of a global process. Is there a reason that we cannot?

And if we keep threeten then it should go in the global list of dependencies. But unless there is a special compatibility reason with other libraries we should use joda.

Reply:
This is due to compatibility reasons. Bigtable uses threeten in its API, hence we have to use it. I'll move this to the global list of dependencies.